src/ADNLPProblems/dixmaan_efgh.jl

```diff
@@ -21,14 +21,14 @@ function dixmaane(;
            sum(T(i / n) * T(δ) * x[i] * x[i + 2 * m] for i = 1:m)
   end
   x0 = 2 * ones(T, n)
-  return ADNLPModels.ADNLPModel(f, x0, name = "dixmaan_efgh"; kwargs...)
+  return ADNLPModels.ADNLPModel(f, x0, name = "dixmaan_e"; kwargs...)
```

Comment:
Thanks. I don't think there should be an underscore though.

Reply:
In the `meta` https://github.com/JuliaSmoothOptimizers/OptimizationProblems.jl/blob/1ef559df2d101557f23e7eed8a2bff09c76be05a/src/Meta/dixmaane.jl#L7
there is no underscore.

Reply:
I'll make a new pull request with no underscore